### PR TITLE
Translation Pipeline PR

### PR DESCRIPTION
**Pull request by guacbot**
This auto-generated PR contains the latest completed translation files downloaded from Transifex.
 Preview: https://docs-staging.datadoghq.com/guacbot/translation-pipeline

### CODE REVIEW

**Comment by urseberry** [comment]
Don't merge this yet - there's an error in one of the files.

> Error building site: "/go/src/github.com/DataDog/documentation/content/ja/agent/guide/network.md:117:6": failed to extract shortcode: shortcode "site-region" has no .Inner, yet a closing tag was provided

File: content/ja/agent/guide/network.md [hunk 1]
```diff
@@ -30,40 +31,129 @@ further_reading:
...
+`gcp-encrypted-intake.logs.datadoghq.eu`<br>
+`http-encrypted-intake.logs.datadoghq.eu`
+
+[1]: /ja/logs/
+[2]: /ja/security/logs/#hipaa-enabled-customers
+[3]: /ja/logs/log_collection/#logging-endpoints
+
+{{< /site-region >}}
+
+Azure Datadog インテグレーションをセットアップするには、Azure サブスクリプションの **Owner** 権限が必要です。セットアップを開始する前に、適切な権限を所有していることをご確認ください。
```
Comment (kayayarai) [inline content/ja/agent/guide/network.md:102]:
```suggestion
{{< site-region region="us3" >}}
```

File: content/ja/integrations/ecs_fargate.md [hunk 2]
```diff
@@ -309,11 +322,95 @@ Datadog の Fluent Bit アウトプットプラグインに組み込まれてい
      }
    }
    ```
+{{< /site-region >}}
+
+Azure Datadog インテグレーションをセットアップするには、Azure サブスクリプションの **Owner** 権限が必要です。セットアップを開始する前に、適切な権限を所有していることをご確認ください。
```
Comment (kayayarai) [inline content/ja/integrations/ecs_fargate.md:327]: {{< site-region region="us3" >}}

Comment (kayayarai) [inline content/ja/integrations/ecs_fargate.md:327]:
```suggestion
{{< site-region region="us3" >}}
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**